sdks/python/apache_beam/io/textio.py

```diff
@@ -83,6 +83,13 @@ def __init__(self, file_pattern, min_bundle_size,
     self._coder = coder
     self._buffer_size = buffer_size
 
+  def display_data(self):
```

Comment:
We normally give as much as possible. I'll add the coder since it's something users may want to find out.